modules/jenkins-plugin/src/main/resources/com/smartbear/ready/jenkins/RemoteTestStarter/config.jelly

```diff
@@ -0,0 +1,81 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry title="Test type" field="testType">
+        <f:select/>
+    </f:entry>
+    <f:entry title="Path to project file" field="pathToProjectFile"
+             description="Specify the path or URL to your test file or directory, or the name of your Repository project">
+        <f:textbox/>
+    </f:entry>
+    <f:section title="XML project options">
+        <f:description>These fields are only applicable for XML/Composite projects</f:description>
+        <f:entry title="Test suite name" field="testSuiteName"
+                 description="The test suite to run from the project.">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="Test case name" field="testCaseName"
+                 description="The test case to run from the project.">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="Active environment" field="environment"
+                 description="The environment to use when running the project.">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="Remote repository" field="repositoryName"
+                 description="Repository on the TestServer where the project is stored.">
+            <f:textbox/>
+        </f:entry>
+    </f:section>
+    <f:section title="Connection Options">
+        <f:description>Only edit these fields if you want to override the default connection settings of the plugin.
+        </f:description>
+        <f:entry title="Server URL" field="serverUrl"
+                 description="The URL to the root of the TestServer, e.g. https://our.testserver.instance:8443/v1.">
```

Comment:
same as below - can we get away with not requiring the user to add the basePath?